plugins/source/gandi/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/gandi/resources/plugin"
+	"github.com/cloudquery/plugin-sdk/serve"
+)
+
+// const sentryDSN = "https://6beffd303fbd4661a7b7c6c2d546b580@o1396617.ingest.sentry.io/6747634" TODO
```

Comment:
```suggestion
const sentryDSN = "https://45b253f7a0794df2bba513eaa930121b@o1396617.ingest.sentry.io/4504214828875776"
```

Here is a new one for the Gandi plugin